docs/customservices.md

```diff
@@ -0,0 +1,38 @@
+# Custom Services
+
+Here is an overview of all custom services that are available within Homer.
+
+## PiHole
+
+Using the PiHole service you can display info about your local PiHole instance right on your Homer dashboard.
+
+The following configuration is available for the PiHole service.
+
+```
+ items:
+      - name: "Pi-hole"
+        logo: "assets/tools/sample.png"
+        # subtitle: "Network-wide Ad Blocking" # optional, if no subtitle is defined, PiHole statistics will be shown
+        url: "http://192.168.0.151/admin"
+        type: "PiHole"
+```
+
+## OpenWeatherMap
+
+Using the OpenWeatherMap service you can display weather information about a given location.
+
+The following configuration is available for the OpenWeatherMap service
+
+```
+items:
+     - name: "Weather"
+       location: "Amsterdam" # your location.
+       locationId: "2759794" # OpenWeatherMap city ID.
```

Comment:
What to you think of
```suggestion
       locationId: "2759794" # Optional: Specify OpenWeatherMap city ID for better accuracy
```

Or something like that. To make it clear that location and locationID are 2 different way to define the same thing.